py/desispec/correct_cte.py

```diff
@@ -342,6 +349,31 @@ def get_cte_params(header, cte_params_filename=None):
     return amp_regions, cte_regions
 
 
+def cte_transfer_amount_regnault(pixel, in_trap, alpha, beta,
+                                 cmax, fin, fout):
+    # leak out
+    transfer_amount = np.clip(fout*cmax*(in_trap/cmax)**alpha, 0, in_trap)
+    # leak in
+    maxin = np.minimum(pixel, cmax - in_trap)
+    alpha = np.clip(alpha, 0, np.inf)
+    beta = np.clip(beta, 0, np.inf)
+    transfer_amount -= np.clip(
+        fin*(1 - in_trap/cmax)**alpha * pixel**beta,
+        0, maxin)
+    return transfer_amount
+
+
+def chi_regnault(param, cleantraces=None, ctetraces=None, uncertainties=None):
+    """Chi loss function for a Regnault transfer function."""
```

Comment:
Please expand docstring to define what inputs mean

Reply:
Updated, resolving.